From 633eba9cf9fec345e73e617d74068822df204156 Mon Sep 17 00:00:00 2001
From: "Ryan X. Charles" <ryan@ryanxcharles.com>
Date: Sat, 18 Jul 2026 11:50:10 -0500
Subject: [PATCH 3/4] Pin reedline path dep in patch.crates-io for Astrohacker
 shell

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 6c10aec70..ee6de25d6 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -479,7 +479,7 @@ bench = false
 # To use a development version of a dependency please use a global override here
 # changing versions in each sub-crate of the workspace is tedious
 [patch.crates-io]
-reedline = { git = "https://github.com/nushell/reedline", branch = "main" }
+reedline = { path = "../reedline" }
 # nu-ansi-term = { git = "https://github.com/nushell/nu-ansi-term.git", branch = "main" }
 
 # Run all benchmarks with `cargo bench`
-- 
2.50.1 (Apple Git-155)

